subscription-service/src/main/resources/db/migration/V0_12__add_expires_at_to_subscription_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE subscription
+ADD expires_at date;
```

Comment:
for dates columns, we use TIMESTAMPTZ type (in NGSI-LD, dates are always represented at UTC time)